src/storage/storage-wrapper.js

```diff
@@ -0,0 +1,21 @@
+import tokens from '@aragon/templates-tokens'
+import { soliditySha3 } from '../web3-utils'
+import storageAbi from './storage-abi.json'
+
+export function testTokensEnabled(network) {
+  return !!tokens[network]
+}
+const getContract = (web3, storageAddr) => {
+  return new web3.eth.Contract(storageAbi, storageAddr)
+}
+
+export default {
+  set: (web3, storageAddr, from, paramName, value) =>
+    getContract(web3, storageAddr)
+      .methods.registerData(soliditySha3(paramName), value)
```

Comment:
soliditySha3 function could be applied just once if we use a constant for that

Reply:
True!